common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -697,6 +708,8 @@ def get_dest_id(self, courselike_key):
                     "Skipping import of Library with id %s, "
                     "since it collides with an existing one", dest_id
                 )
+                if self.status:
+                    self.status.fail(_('Aborting import since a library with this id already exists.'))
```

Comment:
We are not covering use cases with a library that should be a separate task.

Reply:
There is only one case of an error being raised in LibraryImport so I covered that in this PR